### DISCUSSION TITLE
Bed size - X origin greyed out?

### DISCUSSION THREAD

**Discussion by m4R-T1n** [OrcaSlicer/OrcaSlicer#101] [Q&A]
Hello, can someone tell me why its not possible to use the x origin value in the bed size setup? (x greyed out) 
i use different print plates and use it this way to align it to the center.
thanks

**Comment by SoftFever** [comment]
> Hello, can someone tell me why its not possible to use the x origin value in the bed size setup? (x greyed out) i use different print plates and use it this way to align it to the center. thanks

Currently, it doesn't support printers with origin in the bed center(e.g. delta printers). Hence I disabled this option on purpose.

**Comment by m4R-T1n** [comment]
Hello, no my origin is still in the left front corner. i just have a big printbed 350x250 and a 220x220 pei sheet on top in the center.
so i use the offset and pei size in the profile to heve the origin of the pei sheet right. hope you understand
thank you

**Reply by SoftFever** [reply]
Some screenshots will be really helpful.

**Reply by m4R-T1n** [reply]
hope this will help. just a printbed with a pei sheet on top. i want to make 2 printer profiles. first with the offset to "offset 1" und a second for using the printer with the pei sheet with values for "offset 2" in the picture, so i need also the x offset. 
maybe it can be done with 4 areas with non printable sections but thats a mess ^^.
here a picture of the geyed out offset and a sketch from the setup.
thank you

![offset](https://user-images.githubusercontent.com/54242887/210724930-ece99266-1d70-469a-8958-24fc0516f3bc.png)

![printbed offset](https://user-images.githubusercontent.com/54242887/210723581-fc9dbc9d-3c80-4e57-b583-ccdbd16ce1a9.png)

**Reply by SoftFever** [reply]
I see. 

If you are using Klipper, there is a better way to achieve it IMO.
You can leave the bed size as 220x220 as is in the Slicer.
In Klipper config, you can play around with `position_min` for both X and Y axis. e.g. set `position_min`  in [stepper_x] to `-offset_1` (`-65` in your case). Do the same trick for stepper_y. (Don't forget to update other parameters like position_endstop)
This should work ;) 

But the safest and easiest way is probably just to align your PEI sheet with the front left corner of your bed.

**Reply by m4R-T1n** [reply]
thanks, i use marlin on all my printers. i think i will stay with the big size and wait untill the bed model and texture feature work. this way i can use one setting and i will use my bed model to visually see the borders of the pei sheet.
thank you M.
